test/testkit/mocks/prommetricgen/metric_producer.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 const (
-	metricProducerImage = "europe-docker.pkg.dev/kyma-project/prod/samples/telemetry-sample-app:v20250214-f68343ee"
+	metricProducerImage = "europe-docker.pkg.dev/kyma-project/prod/samples/telemetry-sample-app:v20250313-27827878"
```

Comment:
please use "latest" as we had before to directly run into problems if something changed in the sample-app and to keep the setup simple